pkg/authenticator/authenticator.go

```diff
@@ -133,6 +135,13 @@ func (auth *Authenticator) Login() error {
 		return log.RecordedError(log.CAKC029E, err.Error())
 	}
 
+	// ensure client certificate exists before attempting to read it, with a tolerance
+	// for small delays
+	err = waitForCertificate(auth.Config.ClientCertPath, auth.Config.ClientCertTimeout)
```

Comment:
I'd probably name this a bit more specific (eg. `waitForInjectedCertificateFile`) or something similar